config/jobs/kubernetes/enhancements/enhancements-presubmit.yaml

```diff
@@ -7,4 +7,5 @@ presubmits:
       containers:
       - image: golang:1.11
         command:
+        - export PATH=$PATH:$GOPATH/bin
```

Comment:
export is a shell command, this has to be something like:
```yaml
command:
- sh
- c
- "export PATH=$PATH:$GOPATH/bin && ./hack/verify.sh"
```

Reply:
Alternatively, just move it into the verify script?